<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_exit">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>0</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_send">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>621</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_receive">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>621</width>
      <height>161</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_send">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>发送内容：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_receive">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>接收内容：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>板载串口收发系统</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_port_select">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>110</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>串口选择</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_baud">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>160</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_data_bit">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>210</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>数据位</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_parity_check">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>310</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>奇偶校验</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_port_op">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>365</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>串口操作</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear_receive">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear_send">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_send">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_port_op">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>360</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_port_select">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>100</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_baud">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>160</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_data_bit">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>210</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_parity_check">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>310</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_stop_bit">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>260</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>停止位</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_stop_bit">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>260</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
